src/Dns/Dns/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.2.2
+* Added `NAPTR` record type support in cmdlets and updated test recordings.
```

Comment:
Check the PR guidelines to see how to update changelog file. [General Guidelines](https://github.com/Azure/azure-powershell/blob/main/CONTRIBUTING.md#general-guidelines)

Reply:
Made the change as per the guidelines.